### PR TITLE
[TTAHUB-1862] Bug fix with removing recipients

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change

ObjectiveFiles, despite being marked "onDelete: CASCADE" in the model definition, are blocking the removal of recipients, throwing an error when that recipients objectives are removed, causing the entire transaction to be rolled back

## How to test

Fully create an activity report with many recipients, adding one goal and objective with files. Confirm everything is saved properly. To view bug, you can remove a recipient while on ```main``` branch. You should see a 500 error when saving after doing so. On this branch, there is no such error. (You can also comment out lines 1703 - 1709 of /services/goals.js and run the test I added)

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1862

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Comment by thewatermethod** [comment]
This is a branch off prod, that's why the commit history looks a little funky

**Review by nvms — APPROVED** [review]
nice find!

followed the repro steps locally and saw no error

File: src/services/goals.js [hunk 1]
```diff
@@ -1686,13 +1686,35 @@ export async function removeRemovedRecipientsGoals(removedRecipientIds, report)
...
         id: objectivesToDelete,
         onApprovedAR: false,
       },
-      individualHooks: true,
     });
+
+    if (Array.isArray(objectivesToDefinitelyDestroy) && objectivesToDefinitelyDestroy.length > 0) {
+      const objectiveIdsToDestroy = objectivesToDefinitelyDestroy.map((o) => o.id);
+
+      await ObjectiveFile.destroy({
```
Comment (AdamAdHocTeam) [inline src/services/goals.js:1703]: Maybe a dumb question but going forward we will need to make sure to add any other objective tables here?

Comment (thewatermethod) [reply src/services/goals.js:1703]: I don't think that's a dumb question, test verifies we don't need to add topics here but I'll go ahead and add resources so we don't have to worry about it

**Review by AdamAdHocTeam — APPROVED** [review]
Overall looks good and works also verified s

**Pull request merged**